README.md

```diff
@@ -18,6 +18,11 @@ complex molecular systems for molecular dynamics simulations.
 
 To learn more, get started or contribute, check out our [website](http://mbuild.mosdef.org).
 
+### mBuild within the MoSDeF Ecosystem
```

Comment:
For uniformity with the github README outline
![image](https://user-images.githubusercontent.com/11476842/121733183-c900d400-cab8-11eb-9865-8bbfc5f2650a.png)

```suggestion
#### mBuild within the MoSDeF Ecosystem
```